app/views/catalog/search_results.html.erb

```diff
@@ -17,24 +17,24 @@
         </div>
       </div>
     <% end %>
-
   </div>
-
 <% end %>
 
 <% unless @equipment_model_results.empty? %>
-  <% unless @category_results.empty? %>
```

Comment:
It didn't make sense to be there. This was looping through equipment models, so why was there a check for category results? Also, this means that if category_results was empty, it just means "Equipment Model Results" won't show above the equipment model results, which makes no sense.